counter/example.go

```diff
@@ -0,0 +1,59 @@
+package counter
```

Comment:
No, anything with `example` in the path (not just the name of the file) will be excluded from the payload to the user.